dev-haskell/texrunner/texrunner-0.0.1.2-r1.ebuild

```diff
@@ -17,8 +17,6 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-RESTRICT=test # Tests fail. Missing deps?
-
```

Comment:
Did you try the build the ebuild with `FEATURES=test emerge ...`? If it succeeds then the change is fine. But I would guess it will complain about missing files.

Reply:
I tried without, and it worked.
But I'll try again while leaving that line in there and tell what happens.